### DISCUSSION TITLE
Ventura Compatiable Lima Release Date

### DISCUSSION THREAD

**Discussion by a1flecke** [rancher-sandbox/rancher-desktop#3941] [General]
Hello,

Is there a release date for Rancher Desktop that will have a [ventura compatible lima release](https://github.com/rancher-sandbox/rancher-desktop/blame/5ae871ecf41b5766655634fd7725fe553c860619/scripts/dependencies/lima.ts#L29)?  Even if it is just in experimental mode?

Thanks,

Aaron

**Comment by jandubois** [comment]
The current release is compatible with Ventura, but I assume you are looking for one that supports Apple's virtualization framework instead of QEMU. This is currently only available when from CI builds, and it is somewhat tricky to configure manually.

The next release will have an experimental setting to enable this, but there is no specific release date yet, beyond some time in March.

**Reply by a1flecke** [reply]
yes I as referring to a release that supports Apple's virtualization framework.
